admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorerpreviewcontent/template.vue

```diff
@@ -825,12 +825,15 @@ export default {
       this.isOpen = false;
     },
     save() {
+      let promise
       if (this.nodeType === NodeType.OBJECT) {
-        this.saveObject();
+        promise = this.saveObject();
       } else {
-        $perAdminApp.stateAction(`save${this.uNodeType}Properties`, this.node);
+        promise = $perAdminApp.stateAction(`save${this.uNodeType}Properties`, this.node);
         this.edit = false;
```

Comment:
Is it save to release the `this.edit` state without waiting for the promise?

Reply:
Well, it was already like that, so I'd say yes?